src/components/Header/Header.tsx

```diff
@@ -0,0 +1,32 @@
+import { Component } from 'react';
+import ErrorButton from '../ErrorButton/ErrorButton';
+import SearchForm from '../SearchForm/SearchForm';
+import classes from './Header.module.scss';
+
+export default class Header extends Component<{
+  keyword: string;
+  requestCb: (str: string) => void;
+  setErrorStateCb: () => void;
+}> {
+  render() {
+    return (
+      <header>
+        <h1 className={classes.heading}>
+          Welcome to videogames API. This project using the
+          <a href="https://rawg.io/apidocs"></a> RAWG API to display the most
+          <br />
+          high-ranked videogames on Metacritic site by entered keyword. Please,
+          type your request below.
+        </h1>
+        <SearchForm
+          keyword={this.props.keyword}
+          sendRequest={this.props.requestCb}
+        />
+        <ErrorButton
+          text={'Throw an Error'}
+          onClick={this.props.setErrorStateCb}
+        />
+      </header>
+    );
+  }
```

Comment:
use destructuring assignment to make code more readable
![image](https://github.com/getgitgood/react-rss/assets/55094494/7a113c05-c72c-4bd7-9567-4c8441bc2535)